CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 23.02.2
+- Added server configurations feature to enable/disable tracking and networking, this is experimental feature and can cause breaking changes.
```

Comment:
- ! Possible Breaking Changes ! Added 'server configuration' feature that can enable/disable user tracking and networking. This is an experimental feature and might include breaking changes.

Maybe?

Reply:
no need. it's no that the feature introduces breaking changes. but that over the next few releases it could have breaking changes. The idea would be that API "stability" is not guaranteed with this feature